### PR TITLE
remove typescript and update dependencies

### PR DESCRIPTION
**Pull request by erwinmombay**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,46 +6,42 @@
...
-    "eslint": "^3.13.1",
+    "eslint": "4.x",
     "eslint-plugin-ava": "4.2.0",
-    "flow": "0.2.3",
-    "flow-bin": "0.40.0",
-    "gulp": "3.9.1",
+    "gulp": "^3.9.1",
     "gulp-ava": "0.14.1",
-    "gulp-babel": "6.1.2",
+    "gulp-babel": "^8.0.0-beta.2",
```
Comment (rsimha) [inline package.json]: Let's do what `amphtml` does, and use fixed versions. (We can use Renovate on this repo at a later point.)

File: package.json [hunk 2]
```diff
@@ -6,46 +6,42 @@
...
     "gulp-batch": "^1.0.5",
```
Comment (rsimha) [inline package.json]: Use fixed version.

File: package.json [hunk 3]
```diff
@@ -6,46 +6,42 @@
...
     "gulp-eslint": "3.0.1",
-    "gulp-flowtype": "1.0.0",
     "gulp-if": "^2.0.2",
```
Comment (rsimha) [inline package.json]: Use fixed version.

File: package.json (hunk 1 above)
Comment (erwinmombay) [reply package.json]: done.

**Pull request merged**